packages/dataviews/src/components/dataviews-view-config/style.scss

```diff
@@ -29,6 +29,10 @@
 	}
 }
 
+.dataviews-settings-section:has(.dataviews-settings-section__content:empty) {
+	display: none;
+}
```

Comment:
you can define fields that can't be hidden and if all your fields are non hideable, the "properties" section of the view config should be hidden entirely.